src/Microsoft.Repl/Input/KeyHandlers.cs

```diff
@@ -68,50 +68,17 @@ public static void RegisterDefaultKeyHandlers(IInputManager inputManager)
             inputManager.RegisterKeyHandler(ConsoleKey.F23, Unhandled);
             inputManager.RegisterKeyHandler(ConsoleKey.F24, Unhandled);
 
-            inputManager.RegisterKeyHandler(ConsoleKey.Applications, Unhandled);
```

Comment:
What happens if any of these keys get hit?  Does the REPL try to print a character for them?

I'm wondering if in addition to targeting netstandard1.3, we should also target netstandard2.0 and put these removed lines back inside of an `#if NETSTANDARD2_0`.

Reply:
(Or maybe `#if !NETSTANDARD1_3` would be better for future proofing)

Reply:
Yeah, you're right, it attempts to print out `\0`, so that's not ideal. 
Theoretically, this is a problem as they add new keys even beyond 2.0, right? I wonder if we need some other sort of catch all such as if the KeyChar isn't printable, and it's not a registered key handler, then just ignore it. Though defining printable is probably fraught. `IsLetterOrDigit || IsPunctuation || IsSymbol || Environment.NewLine || ' '` probably gets us close... but I'm not sure we can do that reliably.
Thoughts?

Reply:
Does `!char.IsControl(...)` handle these correctly?

Reply:
As discussed offline, `!char.IsControl( ... )` is not sufficient.

I added https://github.com/aspnet/HttpRepl/issues/119 to track the work we'd need to do to handle this. I think it is separate from this PR, this PR just makes it more obvious and important.